docs/export.md

```diff
@@ -0,0 +1,71 @@
+# Export
+
+ROBOT can export details about ontology entities as a table (TSV or CSV). At minimum, the `export` command expects an input ontology (`--input`), a set of columns (`--columns`), and a file to write to (`--export`):
+
+```
+robot export --input nucleus.owl \
+  --columns "CURIE LABEL" \
+  --export nucleus.csv
+```
+
+### Columns
+
+The `--columns` are space-separated special keywords or properties used in the ontology. These should always be enclosed in double quotes.
+
+* **Special Keywords**:
+	* `IRI`: creates an "IRI" column based on the full unique identifier
+	* `CURIE`: creates a "CURIE" column based on the short form of the unique identifier
+	* `LABEL`: creates a "Label" column based on `rdfs:label`
+	* `subclass-of`: creates a "SubClass Of" column based on `rdfs:subClassOf`
+	* `equivalent-classes`: creates an "Equivalent Classes" column based on `owl:equivalentClass`
+	* `subproperty-of`: creates a "SubProperty Of" column based on `rdfs:subPropertyOf`
+	* `equivalent-properties`: creates an "Equivalent Properties" column based on `owl:equivalentProperty`
+	* `disjoints`: creates a "Disjoint With" column based on `owl:disjointWith`
+	* `types`: creates an "Instance Of" column based on `rdf:type` for named individuals
+* **Property CURIES**: you can always reference a property by the short form of the unique identifier (e.g. `oboInOwl:hasDbXref`). ROBOT will attempt to find a label for a CURIE to set as the column header, but if the label is not defined in the ontology, the header will be the CURIE. Any prefix used [must be defined](global/prefixes).
+* **Property Labels**: as long as a property label is defined in the input ontology, you can reference a property by label enclosed in single quotes (e.g. `database_cross_reference`). This label will also be used as the column header.
+
+The first column in the `--columns` list is used to sort the rows of the export. You can change the column that is sorted on by including `--sort <property>`.
+
+All special keyword columns will include both named OWL objects (named classes, properties, and individuals) and anonymous expressions (class expressions, property expressions). When using another object or data property, the values will include both individuals and class expressions (from subclass or equivalent statements) in Manchester syntax. When using an annotation property, the literal value will be returned.
+
+### Including and Excluding Entities
```

Comment:
Instead of these four options we should have a single positive option `--include` that takes a list of "classes", "individuals", "properties", and "anonymous", with a default of `--include "classes individuals". (This list should allow space, comma, or tab separation.) Since `-i` is already used as a short option for `--input`, if you'd rather call this `--export` or something with a free short option, that's fine.

Reply:
Since this list isn't being used to create any headers, does it make sense to allow tab separation? I can see commas or spaces, but the tab separation doesn't make much sense to me.

Reply:
I want to allow tabs in the `--columns` option, and then I think it's more consistent to allow tabs here as well.